CHANGELOG.md

```diff
@@ -188,6 +232,378 @@
 
 **Implemented enhancements:**
 
+- Remove grunt for compress tasks [\#958](https://github.com/emberjs/ember-inspector/issues/958)
```

Comment:
Oops, forgot to do `--since-tag v3.8.0`

Reply:
Should be able to run `yarn changelog` https://github.com/emberjs/ember-inspector/blob/master/package.json#L15